airbyte-integrations/connectors/destination-redshift/src/main/java/io/airbyte/integrations/destination/redshift/RedshiftStagingS3Destination.java

```diff
@@ -105,13 +116,14 @@ public JsonNode toJdbcConfig(final JsonNode config) {
 
   @Override
   public AirbyteMessageConsumer getConsumer(final JsonNode config,
-      final ConfiguredAirbyteCatalog catalog,
-      final Consumer<AirbyteMessage> outputRecordCollector) {
+                                            final ConfiguredAirbyteCatalog catalog,
+                                            final Consumer<AirbyteMessage> outputRecordCollector) {
     final S3DestinationConfig s3Config = getS3DestinationConfig(findS3Options(config));
+    final EncryptionConfig encryptionConfig = EncryptionConfig.fromJson(config.get("loading_method").get("encryption"));
```

Comment:
Could you check whether this line of code won't break the SYNC, in the following scenario:

1. Set Destination Redshift e.g.(0.3.37) version locally.
2. After update to yours (0.3.40).
3. Start the sync and see if it is a success. 

Thanks.